### PR TITLE
Seed file

### PR DESCRIPTION
**Pull request by cou23001**
New seed file with category table

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "seed": "node -r dotenv/config ./scripts/seed.js"
+    "seed": "node -r dotenv/config ./scripts/seed1.js"
```
Comment (MATTALUI) [inline package.json:10]: This won't work because there is no seed1.js

File: scripts/seed.js [hunk 2]
```diff
@@ -73,11 +102,13 @@ const seedProducts = async (client) => {
       CREATE TABLE IF NOT EXISTS group3_products (
         id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
         seller_id UUID NOT NULL,
+        category_id UUID,
```
Comment (MATTALUI) [inline scripts/seed.js:105]: If we set it up this way each product will only have one single category.

**Review by MATTALUI — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cou23001) [reply package.json]: It was corrected

File: scripts/seed.js (hunk 2 above)
Comment (cou23001) [reply scripts/seed.js]: The seed was updated. Each product can have many categories

**Pull request merged**